app/code/Magento/QuoteGraphQl/Model/Cart/AddSimpleProductToCart.php

```diff
@@ -68,15 +68,19 @@ public function execute(Quote $cart, array $cartItemData): void
         $sku = $this->extractSku($cartItemData);
         $qty = $this->extractQty($cartItemData);
         $customizableOptions = $this->extractCustomizableOptions($cartItemData);
+        $downloadableLinks = [];
 
         try {
             $product = $this->productRepository->get($sku);
+            if ($product->getLinksPurchasedSeparately()) {
```

Comment:
Move all of logic to `AddSimpleProductsToCart.php`

Reply:
Changed solution.

Thank you!